### PR TITLE
Updated to latest changes in bolt library

### PR DESCRIPTION
**Pull request by stefanak-michal**
### Description

I updated documentation to follow up on latest changes in bolt library. It contained breaking changes. More details: https://github.com/neo4j-php/Bolt/releases/tag/v5.0.0

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] Documentation improvements
- [x] Other (please describe): Added link for https://github.com/stefanak-michal/memgraph-bolt-wrapper

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [ ] I made corresponding changes to the rest of the documentation
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by stefanak-michal** [comment]
Also please update documentation available in administration of memgraph cloud. I know you have some description there but it is not available publicly for changes.

**Comment by vpavicic** [comment]
> Also please update documentation available in administration of memgraph cloud. I know you have some description there but it is not available publicly for changes.

Will do! Thank you for your contribution!

**Review by vpavicic — APPROVED** [review]

File: memgraph_versioned_docs/version-2.4.2/connect-to-memgraph/drivers/php.md [hunk 1]
```diff
@@ -39,34 +39,48 @@ following code to it:
 <?php
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Create connection class and specify target host and port
+// Create connection class and specify target host and port, default is localhost
 $conn = new \Bolt\connection\Socket();
 // Create new Bolt instance and provide connection object
 $bolt = new \Bolt\Bolt($conn);
+// Set available bolt versions for Memgraph.
+$this->setProtocolVersions(4.1, 4, 3);
```
Comment (tonilastre) [inline memgraph_versioned_docs/version-2.4.2/connect-to-memgraph/drivers/php.md:47]: @stefanak-michal Is this ok? I am not a PHP dev, but should this be:

```php
$bolt->setProtocolVersions(4.1, 4, 3);
```

Where does `$this` comes from?

Comment (stefanak-michal) [reply memgraph_versioned_docs/version-2.4.2/connect-to-memgraph/drivers/php.md:47]: You are correct. Sorry, my mistake.

Comment (vpavicic) [reply memgraph_versioned_docs/version-2.4.2/connect-to-memgraph/drivers/php.md:47]: I'll fix it in the new PR propagating changes to some other pages in docs

**Pull request merged**